shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1505,7 +1505,8 @@ content::JavaScriptDialogManager* WebContents::GetJavaScriptDialogManager(
 }
 
 void WebContents::OnAudioStateChanged(bool audible) {
-  Emit("-audio-state-changed", audible);
+  DCHECK_EQ(audible, IsCurrentlyAudible());
```

Comment:
Why the DCHECK ?

Reply:
I figured it'd be good to ensure that when this event is called it's always the same as `IsCurrentlyAudible()` - happy to remove it though

Reply:
Hmm, I think the check makes sense if they were happening at the content layer before the event was emitted and given it is already being done here https://source.chromium.org/chromium/chromium/src/+/main:content/browser/web_contents/web_contents_impl.cc;l=2369-2373, seems this `DCHECK` is redundant.